lib/misc.js

```diff
@@ -19,6 +19,20 @@ module.exports = (api) => {
     return callback ? wrap : api.common.falseness;
   };
 
+  api.common.cbExtract = (
+    // Exctracts callback and wraps it with api.common.cb
+    // callback is last argument if it's function
+    // otherwise it's api.common.falseness
+    args // arguments
+    // Returns wrapped callback
+  ) => {
+    const argsLast = args[args.length - 1];
+    if (typeof argsLast !== 'function') return api.common.falseness;
```

Comment:
@tshemsedinov, I don't like using `typeof` with function-like syntax because `typeof` isn't function and it can't be saved to variable or passed to function as argument. Therefore operator-like syntax shows non-function nature of `typeof`.
But I will correct in that case because I know that conversation about code-style should be not here